### PR TITLE
 cloud: update navigation steps according to UI

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-support.md [hunk 1]
```diff
@@ -5,51 +5,43 @@ summary: Learn how to contact the support team of TiDB Cloud.
 
 # TiDB Cloud Support
 
-TiDB Cloud offers a free basic support plan for each user and you can upgrade to a paid plan for extended services.
+The **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/) consolidates all the help and support information for TiDB Cloud, such as documentation, interactive tutorials, self-paced training, and support entries.
+
+## Request for PoC, demo, or free trial credits
 
-The information of each support plan is available on the support page of the TiDB Cloud console. To access the page, see [Check or upgrade your support plan](#check-or-upgrade-your-support-plan).
+To request for Perform a Proof of Concept (PoC), demo, or free trial credits, click **?** in the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), and click **Request Support**.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:12]:
```suggestion
To request for a Proof of Concept (PoC), demo, or free trial credits, click **?** in the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), and click **Request Support**.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 2]
```diff
@@ -5,51 +5,43 @@ summary: Learn how to contact the support team of TiDB Cloud.
 
 # TiDB Cloud Support
 
-TiDB Cloud offers a free basic support plan for each user and you can upgrade to a paid plan for extended services.
+The **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/) consolidates all the help and support information for TiDB Cloud, such as documentation, interactive tutorials, self-paced training, and support entries.
+
+## Request for PoC, demo, or free trial credits
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:10]:
```suggestion
## Request PoC, demo, or free trial credits
```

File: tidb-cloud/tidb-cloud-support.md [hunk 3]
```diff
@@ -5,51 +5,43 @@ summary: Learn how to contact the support team of TiDB Cloud.
 
 # TiDB Cloud Support
 
-TiDB Cloud offers a free basic support plan for each user and you can upgrade to a paid plan for extended services.
+The **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/) consolidates all the help and support information for TiDB Cloud, such as documentation, interactive tutorials, self-paced training, and support entries.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:8]:
```suggestion
The **?** icon in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/) consolidates all the help and support information for TiDB Cloud, such as documentation, interactive tutorials, self-paced training, and support entries.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 4]
```diff
@@ -5,51 +5,43 @@ summary: Learn how to contact the support team of TiDB Cloud.
...
 
 - On the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, click **...** in the row of your cluster and select **Get Support**.
 
 - On your cluster overview page, click **...** in the upper-right corner and select **Get Support**.
 
-### Request support for a project or organization
+### Get support for a project or organization
 
-You can request support for a project or organization in either of the following ways:
+To get support for a project or organization, click **?** in the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), and then click **Chat with Us** or **Create New Case**.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-support.md:36]:
```suggestion
To get support for a project or organization, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/), and then click **Chat with Us** or **Create New Case**.
```

File: tidb-cloud/set-up-private-endpoint-connections.md [hunk 5]
```diff
@@ -51,12 +51,8 @@ If you have multiple clusters, you need to repeat these steps for each cluster t
...
 
 1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
-2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
-
-    - If you have multiple projects, switch to the target project, and then click **Admin** > **Network Access**.
-    - If you only have one project, click **Admin** > **Network Access**.
-
-3. Click the **Private Endpoint** tab.
+2. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Admin**.
+3. On the **Admin** page of your project, click **Network Access**, and click the **Private Endpoint** tab
```
Comment (qiancai) [inline tidb-cloud/set-up-private-endpoint-connections.md:55]:
```suggestion
3. On the **Admin** page of your project, click **Network Access**, and click the **Private Endpoint** tab.
```

File: tidb-cloud/configure-maintenance-window.md [hunk 6]
```diff
@@ -57,9 +57,9 @@ You can modify the start time to your preferred time or defer maintenance tasks
...
 
 2. In the left navigation bar, click <svg width="20" height="20" viewBox="0 0 24 24" fill="none" stroke-width="1.5" xmlns="http://www.w3.org/2000/svg"><path d="M12 14.5H7.5C6.10444 14.5 5.40665 14.5 4.83886 14.6722C3.56045 15.06 2.56004 16.0605 2.17224 17.3389C2 17.9067 2 18.6044 2 20M14.5 6.5C14.5 8.98528 12.4853 11 10 11C7.51472 11 5.5 8.98528 5.5 6.5C5.5 4.01472 7.51472 2 10 2C12.4853 2 14.5 4.01472 14.5 6.5ZM22 16.516C22 18.7478 19.6576 20.3711 18.8054 20.8878C18.7085 20.9465 18.6601 20.9759 18.5917 20.9911C18.5387 21.003 18.4613 21.003 18.4083 20.9911C18.3399 20.9759 18.2915 20.9465 18.1946 20.8878C17.3424 20.3711 15 18.7478 15 16.516V14.3415C15 13.978 15 13.7962 15.0572 13.6399C15.1077 13.5019 15.1899 13.3788 15.2965 13.2811C15.4172 13.1706 15.5809 13.1068 15.9084 12.9791L18.2542 12C18.3452 11.9646 18.4374 11.8 18.4374 11.8H18.5626C18.5626 11.8 18.6548 11.9646 18.7458 12L21.0916 12.9791C21.4191 13.1068 21.5828 13.1706 21.7035 13.2811C21.8101 13.3788 21.8923 13.5019 21.9428 13.6399C22 13.7962 22 13.978 22 14.3415V16.516Z" stroke="currentColor" stroke-width="inherit" stroke-linecap="round" stroke-linejoin="round"></path></svg> **Admin**.
```
Comment (hfxsd) [inline tidb-cloud/configure-maintenance-window.md:64]: 这里需要改成新的 svg 吗

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 7]
```diff
@@ -28,13 +28,8 @@ Before adding VPC Peering requests to a region, you need to set a project CIDR f
...
 1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
-2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
-
-    - If you have multiple projects, switch to the target project, and then click **Admin** > **Network Access**.
-    - If you only have one project, click **Admin** > **Network Access**.
-
-3. Click the **Project CIDR** tab.
-
+2. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Admin**.
+3. On the **Admin** page of your project, click **Network Access**, and click the **Project CIDR** tab.
```
Comment (hfxsd) [inline tidb-cloud/set-up-vpc-peering-connections.md:32]:
```suggestion
3. On the **Admin** page of your project, click **Network Access**, and then click the **Project CIDR** tab.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 8]
```diff
@@ -63,12 +58,8 @@ This section describes how to set up VPC peering connections on AWS. For GCP, se
...
 
 1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
-2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
-
-    - If you have multiple projects, switch to the target project, and then click **Admin** > **Network Access**.
-    - If you only have one project, click **Admin** > **Network Access**.
-
-3. Click the **VPC Peering** tab.
+2. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Admin**.
+3. On the **Admin** page of your project, click **Network Access**, and click the **VPC Peering** tab.
```
Comment (hfxsd) [inline tidb-cloud/set-up-vpc-peering-connections.md:62]:
```suggestion
3. On the **Admin** page of your project, click **Network Access**, and then click the **VPC Peering** tab.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 9]
```diff
@@ -213,12 +204,8 @@ Now you have successfully set up the VPC peering connection. Next, [connect to t
...
```
Comment (hfxsd) [inline tidb-cloud/set-up-vpc-peering-connections.md:208]:
```suggestion
3. On the **Admin** page of your project, click **Network Access**, and then click the **VPC Peering** tab.
```

File: tidb-cloud/tidb-cloud-console-auditing.md [hunk 10]
```diff
@@ -17,14 +17,14 @@ TiDB Cloud provides the console audit logging feature to help you track various
...
+1. In the lower-left corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" />, and then click **Console Audit Logging**.
 2. Click **Setting** in the upper-right corner, and then enable console audit logging.
 
 ## Disable console audit logging
 
 To disable console audit logging, take the following steps:
 
-1. In the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> **Organization** > **Console Audit Logging**.
+1. In the lower-left corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" />, and then click **Console Audit Logging**.
 2. Click **Setting** in the upper-right corner, and then disable console audit logging.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-console-auditing.md:28]:
```suggestion
2. Click **Settings** in the upper-right corner, and then disable console audit logging.
```

File: tidb-cloud/tidb-cloud-console-auditing.md [hunk 11]
```diff
@@ -17,14 +17,14 @@ TiDB Cloud provides the console audit logging feature to help you track various
 
 The console audit logging feature is disabled by default. To enable it, take the following steps:
 
-1. In the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> **Organization** > **Console Audit Logging**.
+1. In the lower-left corner of the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" />, and then click **Console Audit Logging**.
 2. Click **Setting** in the upper-right corner, and then enable console audit logging.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-console-auditing.md:21]:
```suggestion
2. Click **Settings** in the upper-right corner, and then enable console audit logging.
```

**Review by shizn — APPROVED** [review]

File: tidb-cloud/tidb-cloud-org-sso-authentication.md [hunk 12]
```diff
@@ -75,7 +75,7 @@ Before enabling Cloud Organization SSO, make sure to inform your members about t
 To enable Cloud Organization SSO, take the following steps:
 
 1. Log in to [TiDB Cloud console](https://tidbcloud.com) as a user with the organization owner role.
-2. In the upper-right corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" /> > **Organization Settings**.
+2. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Organization Settings**.
 3. On the **Organization Settings** page, click the **Authentication** tab, and then click **Enable**.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-org-sso-authentication.md:79]: 界面上没看到 **Authentication** tab
image

File: tidb-cloud/tidb-cloud-password-authentication.md [hunk 13]
```diff
@@ -90,7 +90,7 @@ Two-factor authentication adds additional security by requiring an Authenticator
 
 ### Enable MFA
 
-1. Click <MDSvgIcon name="icon-top-account-settings" /> **Account** in the upper-right corner of the TiDB Cloud console.
+1. Click <MDSvgIcon name="icon-top-account-settings" /> in the lower-left corner of the TiDB Cloud console.
 
 2. Click **Account settings**.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-password-authentication.md:95]:
```suggestion
2. Click **Account Settings**.
```

File: tidb-cloud/tidb-cloud-password-authentication.md [hunk 14]
```diff
@@ -100,7 +100,7 @@ Two-factor authentication adds additional security by requiring an Authenticator
 
 ### Disable MFA
 
-1. Click <MDSvgIcon name="icon-top-account-settings" /> **Account** in the upper-right corner of the TiDB Cloud console.
+1. Click <MDSvgIcon name="icon-top-account-settings" /> in the lower-left corner of the TiDB Cloud console.
 
 2. Click **Account settings**.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-password-authentication.md:105]:
```suggestion
2. Click **Account Settings**.
```

File: tidb-cloud/tidb-cloud-support.md [hunk 15]
```diff
@@ -96,5 +90,5 @@ To check or upgrade your support plan, perform the following steps:
 
 To downgrade your support plan to **Basic**, perform the following steps:
 
-1. On your TiDB Cloud console, click <MDSvgIcon name="icon-top-support" /> **Support** in the upper-right corner of the top navigation bar, and click **View Support Plan**.
+1. In the lower-right corner of your [TiDB Cloud console](https://tidbcloud.com/), click **?** and click **View Support Plan**.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-support.md:93]:
```suggestion
1. In the lower-right corner of your [TiDB Cloud console](https://tidbcloud.com/), click **?** and then click **View Support Plan**.
```

File: tidb-cloud/backup-and-restore.md [hunk 16]
```diff
@@ -165,12 +165,8 @@ To restore your TiDB Dedicated cluster data from a backup to a new cluster, take
 To restore a deleted cluster from recycle bin, take the following steps:
```
Comment (hfxsd) [inline tidb-cloud/backup-and-restore.md:166]: 现在 console 上支持切换 Light 和 Dark，是否也可以在这篇文档里加个描述 https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart

File: tidb-cloud/configure-maintenance-window.md (hunk 6 above)
Comment (qiancai) [reply tidb-cloud/configure-maintenance-window.md]: 这里的描述的确需要改，改到 https://github.com/pingcap/docs/pull/14089/commits/f71e5cb18b7b60ef63739694f14c98dc4fe0535d 这里了

File: tidb-cloud/tidb-cloud-org-sso-authentication.md (hunk 12 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-org-sso-authentication.md:79]: 这个功能在文档提到了是 available upon request

Comment (qiancai) [reply tidb-cloud/tidb-cloud-org-sso-authentication.md:79]: image

**Review by hfxsd — APPROVED** [review]

**Pull request merged**